pkg/store/mockstore/unistore/mock.go

```diff
@@ -30,7 +30,8 @@ import (
 func New(
```

Comment:
prefer we write another `NewClassic` to create uni-store for classic kernel, as `currentKeyspaceID` is always 0 in this case

Reply:
Just to point out that the proper value of `currentKeyspaceID` for non-keyspaced case should be NullKeyspaceID instead of zero